tsconfig.json

```diff
@@ -96,7 +96,7 @@
 
 		/* Completeness */
 		// "skipDefaultLibCheck": true,                      /* Skip type checking .d.ts files that are included with TypeScript. */
-		"skipLibCheck": true /* Skip type checking all .d.ts files. */
+		"skipLibCheck": false /* Skip type checking all .d.ts files. */
```

Comment:
drive-by refactor. Might as well ensure type-coverage isn't implicitly lower due to using `any` from libs.